### DISCUSSION TITLE
How to convert Postman Collection to Folders with .http files?

### DISCUSSION THREAD

**Discussion by ariel-frischer** [rest-nvim/rest.nvim#222] [Q&A]
I haven't dug much into this but is there a way you are exporting postman collections to work with this plugin? Are people doing this one by one?
P.S. Cool plugin I want to get off the postman/insomnia apps clearly.

**Comment by gorillamoe** [comment]
I'm really late, but kulala-fmt can do this.

https://fmt.getkulala.net/

https://github.com/mistweaverco/kulala-fmt?tab=readme-ov-file#postman-collection-to-http
